translations/locale_fr.json

```diff
@@ -0,0 +1,535 @@
+				},
+				"search": {
+					"game_knowledge": {
+						"input_name": "Connaissances du jeu",
+						"label": "Connaissances du jeu"
+					},
+					"logical": {
+						"input_name": "Logique",
+						"label": "Logique"
+					},
+					"showcase": {
+						"fallback": "Choisis une difficulté pour les recherches !",
+						"game_knowledge": {
+							"description": "Pas d'indices de recherche, tu te débrouilles seul·e avec tes connaissances sur le jeu !",
+							"image_alt": "Difficulté de recherche par connaissances du jeu"
+						},
+						"logical": {
+							"description": "La recherche t'indiquera toutes les statistiques disponibles, faisant de MCdle un jeu de logique !",
+							"image_alt": "Difficulté de recherche par logique"
+						}
+					},
+					"title": "Recherche:"
+				},
+				"title": "Difficulté"
+			},
+			"settings_button": {
+				"image_alt": "Bouton des paramètres",
+				"tooltip": "Paramètres"
+			},
+			"title": "Paramètres"
+		},
+		"language": {
+			"title": "Langues",
+			"tooltip": "Langues",
+			"dropdown": {
+				"english": "Anglais",
+				"german": "Allemand",
+				"french": "Français"
+			},
+			"description": {
+				"pre_link": "Officiellement, ce site Web est accessible en anglais et en allemand, mais grâce à la communauté, d'autres langues ont été ajoutées via ",
+				"link": "Github",
+				"post_link": ". Il n'y a aucune garantie que les traductions non officielles soient à jour ou correctes, donc si tu trouves des erreurs, n'hésite pas à ouvrir une pull request !"
+			}
+		}
+	},
+	"pages": {
+		"404": {
+			"compass_alt": "gif de compas",
+			"message": "Tu as rencontré une sortie 404 dans cette grotte...",
+			"return_message": "Utilise ce compas pour retourner à la page d'accueil !"
+		},
+		"index": {
+			"header": "Choisis un mode de jeu !",
+			"links": {
+				"blocks": {
+					"image_alt": "Icône de blocs",
+					"subtitle": "Devine le bloc Minecraft du jour",
+					"title": "Blocs"
+				},
+				"items": {
+					"image_alt": "Icône d'objets",
+					"subtitle": "Trouve l'objet Minecraft du jour",
+					"title": "Objets"
+				},
+				"mobs": {
+					"image_alt": "Icône de créatures",
+					"subtitle": "Découvre la créature Minecraft du jour",
+					"title": "Créatures"
+				},
+				"goal": {
+					"title": "Catalogue MCdle",
+					"image_alt": "Icône objectif"
+				}
+			}
+		},
+		"block": {
+			"title": "Mode de jeu : Devine le bloc du jour !",
+			"sub_title_menus": {
+				"stats": {
+					"title": "Statistiques",
+					"tooltip": "Statistiques"
+				},
+				"streak": {
+					"title": "Transférer les victoires",
+					"tooltip": "Série de victoires"
+				},
+				"help": {
+					"tooltip": "Comment jouer"
+				}
+			},
+			"table": {
+				"header": {
+					"title": "Bloc",
+					"release": "Sortie",
+					"stack_size": "Taille de pile",
+					"tools": "Outil(s)",
+					"blast_resistance": "Résist-ance",
+					"hardness": "Dureté",
+					"flammable": "Inflam_mable",
```

Comment:
Oops, this is a typo!

Reply:
By the way, is there no way to do it automatically using CSS? I'm pretty sure there's a rule about it, but I don't know how to use it. But it would be easier for future translations and style modification.

Reply:
Yes, there is indeed a CSS property for that: `word-break`. But it's best option with default text only produces results like this:
<img width="324" alt="Screenshot 2024-01-07 151611" src="https://github.com/ElitoGame/mcdle-public/assets/43675593/ef42b10f-a6ef-42c1-96a7-5a5d937ce264">
Hence why I prefer to set it in the translations themselves. I remember there are breaking characters, but I think my translation system might not work with them, cause it does automatic escaping and can't add HTML elements. (sadly, I really tried to add that, but the library I use doesn't support it. They say they do, but it doesn't work. And my Issue from 2 months ago has not been answered yet, soooo...